proto/cellar_v2.proto

```diff
@@ -46,6 +46,8 @@ message CellarV2 {
         SetShareLockPeriod set_share_lock_period = 8;
         // Represents function `setupAdaptor(address _adaptor)`
         SetupAdaptor setup_adaptor = 9;
+        // Represents function `setPlatformFee(uint64 platformFee)`
+        SetPlatformFee set_platform_fee = 10;
```

Comment:
Why would we need to adjust the platform fee on v2? Wasn't this only for the new cellar? Was this meant to be in the 2.2 section?